ingestion/src/metadata/ingestion/bulksink/metadata_usage.py

```diff
@@ -348,9 +350,43 @@ def _get_table_life_cycle_data(
         the query being processed.
         """
         try:
-            # Temporary clean up this method to be implemented following the new schema
-            life_cycle = (  # pylint: disable=unused-variable
-                init_empty_life_cycle_properties()
+            life_cycle = LifeCycle()
```

Comment:
if this is not something we want repeated, we can modify it to be used in init

Reply:
It should get initialized to empty for each table so cannot be moved to init